tests/hydrothermal_cooling/screen-output

```diff
@@ -0,0 +1,78 @@
+--------------------------------------------------------------------------
+WARNING: No preset parameters were found for the device that Open MPI
+detected:
+
+  Local host:            blogin2
+  Device name:           i40iw0
+  Device vendor ID:      0x8086
+  Device vendor part ID: 14290
+
+Default device parameters will be used, which may result in lower
+performance.  You can edit any of the files specified by the
+btl_openib_device_param_files MCA parameter to set values for your
+device.
+
+NOTE: You can turn off this warning by setting the MCA parameter
+      btl_openib_warn_no_device_params_found to 0.
+--------------------------------------------------------------------------
+-----------------------------------------------------------------------------
+-----------------------------------------------------------------------------
+
+-----------------------------------------------------------------------------
+-----------------------------------------------------------------------------
+Number of active cells: 256 (on 4 levels)
+Number of degrees of freedom: 5,822 (2,210+297+1,105+1,105+1,105)
+
+Number of mesh deformation degrees of freedom: 594
+   Solving mesh displacement system... 0 iterations.
+*** Timestep 0:  t=0 years, dt=0 years
+   Solving mesh displacement system... 0 iterations.
+   Solving temperature system... 0 iterations.
+   Solving upper system ... 0 iterations.
+   Solving weak_seed system ... 0 iterations.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 0+9 iterations.
+      Relative nonlinear residual (Stokes system) after nonlinear iteration 1: 1
+
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 0+13 iterations.
+      Relative nonlinear residual (Stokes system) after nonlinear iteration 2: 15.086
+
+[blogin2:142522] 1 more process has sent help message help-mpi-btl-openib.txt / no device params found
+[blogin2:142522] Set MCA parameter "orte_base_help_aggregate" to 0 to see all help / error messages
```

Comment:
These lines are HLRN error messages and shouldn't be in screen-output.

Have you tried (in the build dir):
```
make setup_tests
ctest -R hydrothermal_cooling
```
and copying over the results from the build_dir/tests/output-hydrothermal_cooling folder to the tests/hydrothermal_cooling folder? Same for the gnuplot output file.
Then do 
```
make setup_tests
ctest -R hydrothermal_cooling
```
again and the test should succeed :)